client/app/components/cards-list/CardsList.tsx

```diff
@@ -0,0 +1,73 @@
+import { includes, isEmpty } from "lodash";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import Search from "antd/lib/input";
```

Comment:
This will make it use the `Input` component instead of the `Input.Search`. Let's rename it to `Input` here and use `Input.Search` as the component in https://github.com/getredash/redash/blob/d6025f9186d8db8e45b79a6fc4b0afd0855c19fb/client/app/components/cards-list/CardsList.tsx#L42

I think the search is actually not used in the app, but let's keep it as it was in the Class component.

Reply:
Fixed in 83b13dbdb58d43f654223f1da198e387e1bc9ada